docs/tracing/how-to-tracing/feedback-and-annotations/evaluating-phoenix-traces.md

```diff
@@ -153,20 +155,61 @@ We now have a DataFrame with a column for whether each joke is a repeat of a pre
 
 Our evals\_df has a column for the span\_id and a column for the evaluation result. The span\_id is what allows us to connect the evaluation to the correct trace in Phoenix. Phoenix will also automatically look for columns named "label" and "score" to display in the UI.
 
+
 ```python
-eval_df["score"] = eval_df["score"].astype(int)
-eval_df["label"] = eval_df["label"].astype(str)
+from phoenix.client import Client
+
+Client().spans.log_span_annotations_dataframe(dataframe=eval_df, annotation_name="duplicate", annotator_kind="CODE")
+```
+
+You should now see evaluations in the Phoenix UI!
+
+### LLM Eval Example 
+
+Let's use the [Phoenix Evals](https://arize.com/docs/phoenix/evaluation/evals) library to define an LLM-as-a-judge evaluator that classifies jokes as either 
+"nerdy" or "not nerdy." 
+
+```python
+from phoenix.evals import ClassificationEvaluator
+from phoenix.evals.llm import LLM
+
+prompt_template = """
+Determine whether the following joke can be classified as "nerdy" or "not nerdy".
+A nerdy joke is defined as a joke that is related to science, math, or technology.
+
+Joke: {joke}
+"""
```

Comment:
I wonder if we should do something like politeness or something like that - just something slightly towards something that someone might actually "fork"